config/webpack.web.config.js

```diff
@@ -6,7 +6,7 @@ var DIST = path.resolve(__dirname, '../dist');
 var APP  = path.resolve(__dirname, '../src');
 
 module.exports = {
-  entry: path.resolve(APP, 'pact.js'),
+  entry: path.resolve(APP, 'pact-karma.js'),
```

Comment:
Similarly, as `pact-node` became a dependency, webpack pooped its pants and obviously couldn't deal with a few thousand Ruby files now on the path (and of course, this would never work in the browser). It seemed this was the cleanest option, albeit with some duplication.